### DISCUSSION TITLE
Teleport: Distance limit. (Sorry you cant teleport over 128 blocks away!)

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7846] [Enhancement]
Requesting a config option to set a distance limit for teleport between players.
Even better if this is a permisson! so then you can unlock further distances to teleport to!

This also adds consistency with commands like /near that have a DefaultDistance.

**Comment by MrCrazys** [comment]
> Requesting a config option to set a distance limit for teleport between players. Even better if this is a permisson! so then you can unlock further distances to teleport to!
> 
> This also adds consistency with commands like /near that have a DefaultDistance.

Seems like a good idea. Would love to see this!

**Comment by Zrips** [comment]
Should be added with next update

**Reply by MrCrazys** [reply]
> Should be added with next update

Cool, thanks!

**Reply by TomLewis** [reply]
Holy crap thats amazing! Thank you Zrips!
